expat/coverage.sh

```diff
@@ -210,10 +214,11 @@ _main() {
             if ${unicode_enabled} && ! ${with_mingw} ; then
                 continue
             fi
-
-            for xml_context in 0 1024 ; do
-                _build_case
-            done
+	    for xml_attr_info in true false ; do
+	        for xml_context in 0 1024 ; do
+	             _build_case
+		done
+	    done
```

Comment:
Indentation looks off here, see `fi` on the line above. Also, please keep the empty line. Thanks!

Reply:
Addressed review comments in latest patchset